openmetadata-ui/src/main/resources/ui/src/utils/AdvancedSearchUtils.tsx

```diff
@@ -198,7 +199,7 @@ export const generateSearchDropdownLabel = (
           />
         </Typography.Text>
       </Space>
-      {getCountBadge(option.count, 'm-r-sm', false)}
+      {!hideCounts && getCountBadge(option.count, 'm-r-sm', false)}
```

Comment:
Here we wanted to hide the count for all the options whether it has a count or not, but if we hide based on count it won't look good as it will show the count when there is a count but hide the count if it 0,